### ISSUE TITLE
android private dns mode

### ISSUE THREAD

**Issue by Joduai** [censurfridns/client-configs#14]
Is it still possible to use censurfridns / uncensoreddns as a private dns provider hostname on android devices?
I've been trying this long time ago but access had frequent downtimes thus I ceased of using this feature.
Now I was wondering if it's still accessible, and stable for daily use?

ps. is there a guide I could follow to check if DoH really works on a mobile device?

**Comment by samsapti** [comment]
Yes, it seems to still be supported, there's actually a video guide [here](https://github.com/censurfridns/client-configs/blob/main/screenshots/android.mp4), but it's not mention in any `.md` file. Anyways, it works fine for me (uses DoT), you can test it out with something like https://dnsleaktest.com.
